src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts

```diff
@@ -92,6 +97,15 @@ export class HaVoiceCommandDialog extends LitElement {
     this._opened = true;
     await this.updateComplete;
     this._scrollMessagesBottom();
+
+    if (params?.start_listening) {
+      if (this._pipelineId) {
+        await this._getPipeline();
```

Comment:
`await updateComplete` here? As we will already fetch it when `_pipelineId` changes, we now fetch it twice?